src/web/app/app.module.ts

```diff
@@ -46,6 +46,11 @@ let routes: Routes = [
         component: PublicPageComponent,
         loadChildren: () => import('./user-join-page.module').then((m: any) => m.UserJoinPageModule),
       },
+      {
+        path: 'createaccount',
+        component: PublicPageComponent,
+        loadChildren: () => import('./user-create-account-page.module').then((m: any) => m.UserCreateAccountPageModule),
```

Comment:
I'm really not sure why we need an entire new page for this, especially if you look at the duplications. Why not just reuse the user join page? It shouldn't be difficult to differentiate how the page is processed by using some form of query parameter combination.